BrainPortal/app/models/cluster_task.rb

```diff
@@ -2180,6 +2180,9 @@ def singularity_commands(command_script)
 
 # Make sure it is executable
 chmod 755 #{singularity_wrapper_basename.bash_escape}
+# Other should have executable right on all components
+# of the path in order to be mounted by singularity.
+chmod o+x . .. ../.. ../../..
```

Comment:
This seems scary to me if it is not put in the absolute right place.  Just flagging to make sure you determined it wasn't so scary.